test/test_multiprocessing.py

```diff
@@ -22,6 +22,16 @@
 TEST_MULTIGPU = TEST_CUDA_IPC and torch.cuda.device_count() > 1
 
 
+class SubProcess(mp.Process):
+    def __init__(self, tensor):
+        super(SubProcess, self).__init__()
+        self.tensor = tensor
+        self.daemon = True
+
+    def run(self):
+        self.tensor.add_(3)
```

Comment:
Why does it have to be in the global scope?

Reply:
It's not supported for mutiprocessing in Windows if the class is defined in the function.